src/reynir_correct/main.py

```diff
@@ -341,17 +343,43 @@ def sentence_stream() -> Iterator[List[CorrectToken]]:
             )
             for ann in a
         ]
+        if args.text:
+            arev = a
+            arev.sort(key=lambda ann: (ann.start, ann.end), reverse=True)
+            if sent.tree is None:
+                # No need to do more, no grammar errors have been checked
+                print(cleaned, file=args.outfile)
+            else:
+                # We know we have a sentence tree, can use that
+                cleantoklist: List[CorrectToken] = toklist
```

Comment:
Again, this just creates a second reference to the same list. To keep toklist intact, use:

`cleantoklist: List[CorrectToken] = toklist[:]`

Reply:
Done